package.json

```diff
@@ -79,7 +79,7 @@
     "solhint": "^1.1.8",
     "solidity-sha3": "^0.4.1",
     "tiny-promisify": "^1.0.0",
-    "truffle": "4.0.3",
```

Comment:
anything we should know about how or why this was versioned down in the first place?